rpc/src/v1/types/trace.rs

```diff
@@ -252,14 +255,40 @@ pub enum CallType {
 	StaticCall,
 }
 
-impl From<vm::CallType> for CallType {
-	fn from(c: vm::CallType) -> Self {
+impl From<vm::ActionType> for CallType {
+	fn from(c: vm::ActionType) -> Self {
 		match c {
-			vm::CallType::None => CallType::None,
-			vm::CallType::Call => CallType::Call,
-			vm::CallType::CallCode => CallType::CallCode,
-			vm::CallType::DelegateCall => CallType::DelegateCall,
-			vm::CallType::StaticCall => CallType::StaticCall,
+			vm::ActionType::Call => CallType::Call,
+			vm::ActionType::CallCode => CallType::CallCode,
+			vm::ActionType::DelegateCall => CallType::DelegateCall,
+			vm::ActionType::StaticCall => CallType::StaticCall,
+			vm::ActionType::Create => CallType::None,
+			vm::ActionType::Create2 => CallType::None,
+		}
+	}
+}
+
+/// Create type.
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum CreateType {
+	/// None
+	None,
+	/// Create
+	Create,
+	/// Create2
+	Create2,
+}
+
+impl From<vm::ActionType> for CreateType {
```

Comment:
Instead of doing `From` maybe better to implement it like this:
```
impl CreateType {
  pub fn new(action_type: vm::ActionType) -> Option<Self> {
    ...
  }  
}
```